modules/runtime-info-ui-ng/src/main/resources/ui/rest_docs.js

```diff
@@ -33,13 +33,18 @@ $(document).ready(function($) {
   $('input').keyup(search);
 
   $.getJSON('/info/components.json', function(data) {
+    var docs = $('#docs'),
+        tpl = $('#template').html();
     $.each(data, function(section) {
       if ('rest' == section) {
         data.rest.sort((a,b) => a.path > b.path ? 1 : -1);
         $.each(data.rest, function(i) {
-          $('#docs').append('<li><a href="/docs.html?path=' + data.rest[i].path + '">'
-              + '<span>' + data.rest[i].path + '</span>'
-              + data.rest[i].description + '</a></li>');
+          let path = data.rest[i].path,
```

Comment:
According to https://caniuse.com/#search=let, this will not work on Opera mini (2.5% market share). No strong opinion on that, but seems that ECMA2015 is still not fully supported on all browsers out there.

Maybe just re-use data.rest[i].path two times to avoid variable hoisting and about service=$(tpl)? 

As said, no strong opinion here.

Reply:
Other UI's of Opencast already require ES2015, including the Paella player. And I don't espect many Opera Mini users to complain about this UI ;-P

But in general, this might be something we want to bring up in the community.

Reply:
Some example of ED2015 usage:
- [REST docs](https://github.com/opencast/opencast/blob/develop/modules/runtime-info-ui-ng/src/main/resources/ui/rest_docs.js#L38)
- [Paella](https://github.com/opencast/opencast/blob/develop/modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.loader/04_datadelegates.js#L201)

Reply:
As said, the Paella player transpiles this into ES5 using traceur.

Reply:
And the REST docs example was introduced by you 16 days ago ;-)

Reply:
> As said, the Paella player transpiles this into ES5 using traceur.

 If you insist: Please show me the code that calls the transpiler.

Reply:
Here you are: https://github.com/polimediaupv/paella/blob/develop/gulpfile.js#L88

Admittedly, their build task generates both a ES5 and ES6 variant of the player. Seems Opencast uses the ES5, at least our watch.html is correct (see https://github.com/opencast/opencast/blob/develop/modules/engage-paella-player/src/main/paella-opencast/ui/watch.html#L8).

Don't get me wrong: I've stumbled over the let statement because I fixed an error in the Paella build pipeline a while ago which was about replacing their uglifier library through a ES6-capable uglifier. The old one failed because of let statement ;-)

In general, ES6 would be desirable. Just not clear to me whether that will cause problems or not. And it seems that you have silently introduces ES6 in Opencast with some of your recent pull requests.

Reply:
As mentioned before:

> I'm talking about the bindings and they don't. They require ES2015

That's also the code I linked. It's in Opencast. Hence, please show me the code in Opencast that transpiles *our* code to ES5.

Reply:
Oh, now I got the point. In that case: @rrolf Have you been aware that Paella puts quite some effort to be compatible to ES5 when you updated the bindings? Or is this an oversight?

Reply:
@staubesv, let us not start a discussion here. If you are concerned about this, please bring this to list and maybe the technical meeting.

Back to topic: Do you now have a problem with this pull request containing `let`?

Reply:
@lkiesow The Opencast bindings also go through Traceur, so the official player is ES5.

https://github.com/opencast/opencast/blob/develop/modules/engage-paella-player/gulpfile.js#L11 copies the files into the folder where the external Paella standalone is copied and https://github.com/opencast/opencast/blob/develop/modules/engage-paella-player/gulpfile.js#L35 uses the build task of that external Paella player which will generate paella_player.js (or similar name) that is ES5 (including the Opencast bindings).

Note that Valencia still requires ES5 because some of their users rely on Internet Explorer support being working.

I can confirm that we also have institutions that still rely on Internet Explorer support in the player.

But anyway: Opencast Paella does support Internet Explorer (at least inofficially).

I don't see a problem for this PR as the REST docs are targeting developers which can be expected to not use IE.